scripts/evolving.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$data = file_get_contents('/var/www/log/time_to_evolve.txt');

```

Comment:
Should be there a check for if the file exists?

Reply:
This file is created in the other script that does the search. All of these are pre-requisites to run this script.